Code/Launcher/Game/GameLauncher.cpp

```diff
@@ -12,6 +12,8 @@
 #define LAUNCHER_BANNER "C1-Launcher Game " PROJECT_VERSION_STRING
 #define DEFAULT_LOG_FILE_NAME "Game.log"
 
+bool g_isCrysisMP = false;
```

Comment:
I find this variable name confusing because multiplayer is supported by all Crysis versions. Well, except Warhead. What about:

```suggestion
bool g_isCrysisBetaBuild = false;
```